### PR TITLE
Accessibility on RRs

### LINKED ISSUE #1139: Read receipts not accessible with VoiceOver
### Steps to reproduce

Turn on VoiceOver and open an conversation, so you can read the messages.
When you put the VoiceOver cursor on the read receipt, (that round checkmark) VoiceOver doesn't give a clear description of what that circle means.

#### What did you expect?

I expected the following behavior:
(not read) =  "Message delivered"
(read by @harveyharvest:matrix.org with display name "Harvey") = "Read by Harvey"
(read by multiple users with display names "Stan" "Leo" "Harvey") = "Read by Stan, Leo and Harvey."

#### What happened instead?

I found an "unlabeled" image. This is not useful for Visually impaired users.

### Your phone model

iPhone 10s

### Operating system version

iOS 16.5

### Application version

1.1.3

### Homeserver

matrix.org

### Will you send logs?

No

### PR DESCRIPTION
**Pull request by Velin92**
fixes #1139

### CODE REVIEW

File: ElementX/Resources/Localizations/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -179,6 +184,7 @@
 "common_voice_message" = "Voice message";
 "common_waiting" = "Waiting…";
 "common_waiting_for_decryption_key" = "Waiting for this message";
+"common.seen_by" = "Seen by";
```
Comment (pixlwave) [inline ElementX/Resources/Localizations/en.lproj/Localizable.strings:187]: This one seems related to this PR, but unused?

File: ElementX/Sources/Screens/RoomScreen/View/Supplementary/TimelineReadReceiptsView.swift [hunk 2]
```diff
@@ -40,12 +40,52 @@ struct TimelineReadReceiptsView: View {
...
+            }
+            
+            // Plurals with string arguments aren't generated correctly so we need to use this
+            // https://github.com/SwiftGen/SwiftGen/issues/1089
+            return L10n.tr("Localizable", "a11y_read_receipts_multiple_with_others", list, remaining)
+        }
+        return ""
+    }
+    
+    private func getDisplayName(at index: Int) -> String {
```
Comment (pixlwave) [inline ElementX/Sources/Screens/RoomScreen/View/Supplementary/TimelineReadReceiptsView.swift:86]:
```suggestion
    private func displayName(at index: Int) -> String {
```

File: ElementX/Sources/Screens/RoomScreen/View/Supplementary/TimelineReadReceiptsView.swift [hunk 3]
```diff
@@ -40,12 +40,52 @@ struct TimelineReadReceiptsView: View {
...
+            return L10n.a11yReadReceiptsSingle(getDisplayName(at: 0))
+        } else if timelineItem.properties.orderedReadReceipts.count <= displayNumber {
+            let limit = timelineItem.properties.orderedReadReceipts.count - 1
+            var list = ""
+            for index in 0..<limit {
+                list += "\(getDisplayName(at: index))"
+                if index != limit - 1 {
+                    list += ", "
+                }
+            }
```
Comment (pixlwave) [inline ElementX/Sources/Screens/RoomScreen/View/Supplementary/TimelineReadReceiptsView.swift:67]: Might be a bit more succinct to do this here and below.
```suggestion
            (0..<limit).map { getDisplayName(at: $0) }.formatted(.list(type: .and, width: .narrow))
```

**Review by pixlwave — APPROVED** [review]
👍

File: ElementX/Resources/Localizations/en.lproj/Localizable.strings (hunk 1 above)
Comment (Velin92) [reply ElementX/Resources/Localizations/en.lproj/Localizable.strings:187]: maybe I added it by mistake? Don't remember it

**Pull request merged**